plugins/inventory/opennebula.py

```diff
@@ -0,0 +1,227 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020, FELDSAM s.r.o. - FeldHost™ <support@feldhost.cz>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+    name: opennebula
+    plugin_type: inventory
+    author:
+        - Kristian Feldsam
+    short_description: OpenNebula inventory source
+    version_added: "2.10"
+    extends_documentation_fragment:
+        - constructed
+    description:
+        - Get inventory hosts from OpenNebula cloud.
+        - Uses an YAML configuration file ending with either I(opennebula.yml), I(opennebula.yaml), I(one.yml)
+        - or I(one.yaml) to set parameter values.
+        - Uses I(api_authfile), I(~/.one/one_auth), or C(ONE_AUTH) pointing to a OpenNebula credentials file
+    options:
+        plugin:
+            description: Token that ensures this is a source file for the 'opennebula' plugin.
+            type: string
+            required: True
+            choices: [ opennebula ]
+        api_url:
+            description:
+              - URL of the OpenNebula RPC server.
+              - It is recommended to use HTTPS so that the username/password are not
+              - transferred over the network unencrypted.
```

Comment:
```suggestion
              - It is recommended to use HTTPS so that the username/password are not
                transferred over the network unencrypted.
```

Reply:
and to use an active voice instead of passive is considered as a best practice (according to the ansible doc guide)